### PR TITLE
[DOCS] Improve readability of INSTALL.md

### PR DESCRIPTION
**Pull request by zainhaq-h2o**
This PR improves the readability of the `INSTALL.md` file.

### CODE REVIEW

File: docs/INSTALL.md [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
 sudo apt-get -y install nvidia-headless-535-server nvidia-fabricmanager-535 nvidia-headless-no-dkms-535-servers nvidia-utils-535-server
 ```
 
-With this one does not need to use nvidia developer downloads below
+Note that if you run the preceding commands, you don't need to use the NVIDIA developer downloads in the following section.
 
-### Installing CUDA Toolkit
+### Install CUDA Toolkit
 
 E.g. CUDA 12.1 [install cuda coolkit](https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&Distribution=Ubuntu&target_version=22.04&target_type=deb_local)
```
Comment (zainhaq-h2o) [inline docs/INSTALL.md:18]: @pseudotensor This link points to CUDA Toolkit 12.2 - should this line be changed to say "E.g. CUDA 12.2"?

Comment (pseudotensor) [reply docs/INSTALL.md:18]: It's unversioned latest.  Can change "CUDA 12.1" -> "Latest CUDA"

Comment (zainhaq-h2o) [reply docs/INSTALL.md]: Done, thanks
https://github.com/h2oai/h2ogpt/pull/971/commits/1bd7f1ae8cfd434ee73b1c8d4bc0ef48ad33e0f5

**Review by pseudotensor — APPROVED** [review]
thanks!!

**Pull request merged**